### DISCUSSION TITLE
Add a option for "Raw values" in the permission rules

### DISCUSSION THREAD

**Discussion by Attacler** [directus/directus#10332] [Feature Requests]
Right now we cannot fill in a date filter like $NOW:
![image](https://user-images.githubusercontent.com/42644889/144854834-6d5f18c4-3c3f-493b-b730-bff9e4f7b7ee.png)

So my idea is to slam in a option:
![image](https://user-images.githubusercontent.com/42644889/144855266-b186edd1-1d41-4b4d-9d65-5767cd56ea3c.png)
If you select date (or the type of the field) it will give you the default option like before.
If you select raw it will give you the option to fill in any string, for example $NOW.

Sorry for the dutch translation btw

**Comment by benhaynes** [comment]
I believe this is very similar to the fix that we have in the works. 👍

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
